osu.Game/Localisation/BeatmapOverlayStrings.cs

```diff
@@ -28,6 +28,11 @@ public static class BeatmapOverlayStrings
         /// </summary>
         public static LocalisableString UserContentConfirmButtonText => new TranslatableString(getKey(@"understood"), @"I understand");
 
+        /// <summary>
+        /// "Toggling this filter is disabled in this platform."
+        /// </summary>
+        public static LocalisableString FeaturedArtistsDisabledTooltip => new TranslatableString(getKey(@"featured_artists_disabled_tooltip"), @"Toggling this filter is disabled in this platform.");
```

Comment:
this doesn't do anything to explain the "why". you can already see it's disabled by it being greyed out.

maybe a tooltip is fine, but the text needs rethinking.

also an aside since we'll be changing it, but it should read "on this platform", not "in".

Reply:
I've added it because it states that the filter is disabled at a platform-level, better than not say anything and have the user think they disabled a setting somewhere or the game is bugged. An explanation would be good if one can be thought of, but at least mentioning that it's happening at a platform-level avoids users thinking it's a bug.